test/Core/Common/Context.hs

```diff
@@ -5,22 +5,53 @@ import Juvix.Library
 import qualified Test.Tasty as T
 import qualified Test.Tasty.HUnit as T
 
-nestedRecord :: Context.T term ty sumRep
-nestedRecord =
-  Context.fromList
-    [ ( "foo",
-        Context.Record
-          (Context.fromList [("a", Context.Record (Context.T mempty) Nothing)])
-          Nothing
-      )
-    ]
-
-memptyTest :: Context.Definition Natural Natural Natural
-memptyTest =
-  foldr
-    const
-    (Context.Def Nothing Nothing 1 Context.default')
-    (fmap identity (Context.fromList []))
-
-lookupTest :: Maybe (Context.Definition term ty sumRep)
-lookupTest = Context.lookup "foo.a" nestedRecord
+-- nestedRecord :: Context.T term ty sumRep
```

Comment:
Why are these commented?

Reply:
These examples don't really work as written anymore, have to induce it another way, hence the comments and the extra functions below this

Reply:
Yes, modules are records afterall, so they have to be the same!